pkg/network/controllers/vm_test.go

```diff
@@ -321,11 +327,13 @@ func isSyncErrorType(e error) bool {
 	return errors.As(e, &errWithReason)
 }
 
-func plugNetworkInterface(vm *v1.VirtualMachine, netName string) *v1.VirtualMachine {
+func plugNetworkInterface(vm *v1.VirtualMachine, ifaceToPlug v1.Interface) *v1.VirtualMachine {
 	vm.Spec.Template.Spec.Domain.Devices.Interfaces = append(
 		vm.Spec.Template.Spec.Domain.Devices.Interfaces,
-		libvmi.InterfaceDeviceWithBridgeBinding(netName),
+		ifaceToPlug,
 	)
+
+	netName := ifaceToPlug.Name
```

Comment:
nit: Can be inline.

Reply:
Since it is used twice I find it more readable than:
```go
*libvmi.MultusNetwork(ifaceToPlug.Name, ifaceToPlug.Name+"-nad"),
```